cmd/system-probe/modules/network_tracer.go

```diff
@@ -143,6 +143,7 @@ func (nt *networkTracer) GetConnections(req *connectionserver.GetConnectionsRequ
 		cs.Conns = rest
 	}
 
+	network.Reclaim(cs)
```

Comment:
you should put it as a `defer` right after checking the error from `GetActiveConnections`
we can abort in the middle in the loop

Reply:
done